tests/integration_tests/example_workspaces/pkl_project/MODULE.bazel

```
@@ -0,0 +1,59 @@
+# Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+bazel_dep(name = "rules_pkl", version = "1.0")
+local_path_override(
+    module_name = "rules_pkl",
+    path = "../../../..",
+)
+
+bazel_dep(name = "aspect_bazel_lib", version = "2.5.3")
+
+pkl = use_extension("@rules_pkl//pkl:extensions.bzl", "pkl")
+pkl.project(
+    name = "mypkl",
+    pkl_project = "@//:PklProject",
+    pkl_project_deps = "@//:PklProject.deps.json",
+
+    # Pass necessary environment variables during the evaluation of the PklProject file.
+    extra_flags = [
+        "--env-var",
+        "PACKAGE_NAME=mypackage",
+    ],
+
+    # Simpler way to pass environment variables during the evaluation of the PklProject file.
+    environment = { 
+        "PACKAGE_VERSION": "1.0.0",
```

Coverage could be slightly more extensive
```suggestion
        "PACKAGE_VERSION": "1.0.0",
        "RANDOM_VARIABLE": "Alternative '\"😀 content!!&;"
```